docs/src/main/sphinx/release/release-385.md

```diff
@@ -0,0 +1,37 @@
+# Release 385 (dd June 2022)
+
+## General
+
+* Introduce `json_array` and `json_object` [JSON functions](/functions/json). ({issue}`9081`)
+* Support all types that can be cast to `varchar` as parameters for the JSON
+  path. ({issue}`12682`)
+* Allow `CREATE TABLE LIKE` a table from a different catalog when explicitly
+  excluding properties. ({issue}`3171`)
+* Reduce `Exceeded limit of N open writers for partitions` errors when
+  fault-tolerant execution is enabled. ({issue}`12721`)
+
+## Delta Lake connector
+
+* Add support for the `appendOnly` field. ({issue}`12635`)
+* Add support for column comments when creating a table or a column. ({issue}`12455`, {issue}`12715`))
+
+## Hive connector
+
+* Allow cancelling a query on a transactional table if it is waiting for a lock. ({issue}`11798`)
+* Propagate compression codec selected by `hive.compression-codec` config
+  property or `compression_codec` session property to Avro file writer. ({issue}`12639`)
+
+## Iceberg connector
+
+* Improve query performance when a table consists of many small files. ({issue}`12579`)
+* Improve query performance when performing a delete or update. ({issue}`12671`)
+* Add support for time travel. ({issue}`10258`)
```

Comment:
I think I'd like to document what time travel is and reference that here, rather than try to explain how to use time travel in release notes. Going to get a PR for this up ASAP.

Reply:
I would rephrase as "Add support for [VERSION | TIMESTAMP] AS OF clause" or something similar. There's not such a thing as "time travel" in the SQL language or spec.

Also, we already support querying older snapshots of a table by referencing special table names.

Reply:
I don't see anything there that can be linked to...